### DISCUSSION TITLE
Windows系统下DialTLSContext错误问题

### DISCUSSION THREAD

**Discussion by coldjelly** [mack-a/v2ray-agent#770] [General]
用mack-a大佬的脚本搭了VLESS-Vision-uTLS-REALITY，搭完以后在Windows系统下使用就会报以下错误：

2023/08/31 00:03:59 127.0.0.1:49882 accepted //www.google.com:443 [http >> proxy]
REALITY localAddr: 192.168.0.105:49883	DialTLSContext
REALITY localAddr: 192.168.0.105:49883	DialTLSContext
……
端口号会持续增加，不断报DialTLSContext

服务端和客户端的Xray-core均为1.8.4版本。使用V2RayN和NekoRay客户端均有同样错误。

奇怪的是，同样的网络环境，Android系统用V2RayNG使用该配置则无任何问题。

之后又尝试了另一个一键脚本：https://github.com/jinwyp/one_click_script/blob/master/README2_CN.md 使用这个脚本搭建的VLESS-Vision-uTLS-REALITY没有任何问题，在Windows环境下也可以正常使用。

对比了两个脚本生成的配置，可以正常使用的后者在配置上多了伪装域名(host)和ShortId。这两个配置项都是可选的。伪装域名(host)我填的是服务端VPS的IP，我理解对客户端不同操作系统的使用也不会有什么影响。

百思不得其解，权且记录下来，如果大佬有空可以看看。

**Comment by mack-a** [comment]
有空看下

**Comment by mack-a** [comment]
ShortId加上了可以试一下

**Comment by mc1423813868** [comment]
我也是一样的问题，折腾好久！！！
